examples/layout/main.go

```diff
@@ -126,9 +126,9 @@ var (
 	list = lipgloss.NewStyle().
 		Border(lipgloss.NormalBorder(), false, true, false, false).
 		BorderForeground(subtle).
-		MarginRight(2).
+		MarginRight(1).
 		Height(8).
-		Width(columnWidth + 1)
```

Comment:
we don't need to do this anymore because the `+1` before was to compensate for the border on one side